tests/testExamples.m

```diff
@@ -140,6 +142,17 @@ function testLiveScriptExamples(testCase, exampleLiveScriptFile)
             close all;
         end
 
+        function testTutorialFiles(testCase, tutorialFiles)
+            % runs the example live script file
+            testExamples.setupCurrentDir(testCase, tutorialFiles) 
+            problem1 = load('twoContrastExample.mat');
+            problem1 = problem1.problem; 
+            evalc(tutorialFiles);
+            problem2 = load('twoContrastExample.mat');
+            problem2 = problem2.problem;
+            testCase.verifyEqual(problem1, problem2, 'twoContrastExample is not correctly updated');           
+        end
```

Comment:
What happens if this test fails after the `evalc`, and is then re-run. Presumerably, the `problem` saved to `twoContrastExample.mat` is incorrect and the test will always fail?

Reply:
The original 'twoContrastExample.mat' is never changed as the test is run in a temp directory